### DISCUSSION TITLE
Crash on launch, Android 4.4.4

### DISCUSSION THREAD

**Discussion by yvant** [airsdk/Adobe-Runtime-Support#2554] [Q&A] [answered]
Hi, 

Using Air SDK 50.2.x, my game doesn't run on older devices and crashes after launch. It works fine when I use SDK 50.1.1.2.
I wasn't able to identify the source of the problem, but here's a log:
[log-crash-at-start-4.4.4.txt](https://github.com/airsdk/Adobe-Runtime-Support/files/11093709/log-crash-at-start-4.4.4.txt)
Do you see anything in there that could help?

Thanks!

**Comment by ajwfrost** [comment, marked as answer]
Hi
Thanks for this, someone reported something similar. The log file is full of cases where the Android system is trying to resolve the API calls, I believe we had isolated the issue here because there was a call to an API that we were making that was only introduced from Android API level 21, but we had missed putting this into a conditional execution statement.. so I think this will be fixed in our next release.
thanks
